### DISCUSSION TITLE
Is there a way to check the presense of special arguments?

### DISCUSSION THREAD

**Discussion by Wohlstand** [WeblateOrg/weblate#9379] [Q&A] [answered]
Hello!

Is possible to have a check that requires presense of specific keywords in the translation? I checking the documentation, and can't find any related stuff.

For example, there is a translation of a formatted string like `"We have %1 cats and %2 dogs in the house."` or `"We saw {0} birds are flying in the sky. Someone stole {1} apples from my bag while I counted crows!"`. In these examples are special arguments of different formats, like `%1` (Qt Linguist) and `{1}` (fmt.h library). If these arguments got missing, libraries may throw exceptions or report an error. So, it would be cool to also check the presence of arguments if they are at the source.

**Comment by nijel** [comment, marked as answer]
Yes, see https://docs.weblate.org/en/latest/user/checks.html#formatted-strings

**Reply by Wohlstand** [reply]
Super!
